### ISSUE TITLE
SEO - url improvments

### ISSUE THREAD

**Issue by Pauline0504** [FeiyChina/feiy#404]
Would need to improve urls of the website to optimize SEO:

For the organisations, jobs and events: Can it be automatic ?
http://www.feiy.co/organizations/11 -> http://www.feiy.co/organizations/Relabs
http://www.feiy.co/organizations/19/events/67 -> http://www.feiy.co/events/+eventname (maybe would need to limitate the event title size?)
http://www.feiy.co/organizations/19/jobs/17 -> http://www.feiy.co/ELG/HR-assistant (+org+job position)

For the blog: I don't know what is the right recommandation? 
1/ Chose ourselves? 2/ Also make it automatic and just put the name of the article? 
http://www.feiy.co/articles/24

**Comment by xnagpa** [comment]
Yes, we can do everything automatically. I need to write a script, which will fill friendly_url with organization/article/event title.  I'll get rid of chinese letters and leave only latin alphabet.

**Comment by Pauline0504** [comment]
Awesome
